addon/components/universal-export/component.js

```diff
@@ -131,6 +131,7 @@ export default Component.extend({
             'Export completed',
             this._toastConfig
           );
+          this.toggleProperty('hideInfluencerNetwork')
```

Comment:
This should not be scoped to client with network based price ?

Reply:
Hmm :thinking: 
@SimonBoxus is there a scope to consider here? I'll need the name of the scope if so